packages/terra-form-input/tests/jest/InputField.test.jsx

```diff
@@ -20,8 +20,8 @@ it('should render a InputField with props', () => {
       hideRequired
       inputAttrs={{
         name: 'test',
-        type: 'number',
       }}
+      type="number"
```

Comment:
Why was this changed? It is valid to have `type` as part of `inputAttrs` imo.

Reply:
To keep it align with the code changes made for InputField. Now Input Field has type as first class prop. Since type prop will be having the priority over type specified within Inputattrs modified this example.

Reply:
We should have a test for both scenarios.

Reply:
+1. Please add tests to cover all possible scenarios.

Reply:
aded tests to cover inputAttrs and type prop scenario : [code](https://github.com/cerner/terra-core/pull/2337/commits/c150a05b7b97863658fa3ce7ea0a07b7158fe7a3)